examples/DGmethods/ex_001_dycoms.jl

```diff
@@ -58,81 +58,96 @@ eprint = {https://doi.org/10.1175/MWR2930.1}
-  ρ_sfc::DT     = 1.22
-  P_sfc::DT     = 1.0178e5
-  T_BL::DT      = 285.0
-  T_sfc::DT     = P_sfc/(ρ_sfc * Rm_sfc);
+  epsdv::FT     = molmass_ratio
+  q_tot_sfc::FT = 8.1e-3
+  Rm_sfc::FT    = gas_constant_air(PhasePartition(q_tot_sfc))
+  ρ_sfc::FT     = 1.22
+  P_sfc::FT     = 1.0178e5
+  T_BL::FT      = 285.0
+  T_sfc::FT     = P_sfc/(ρ_sfc * Rm_sfc);
   
-  q_liq::DT      = 0
-  q_ice::DT      = 0
-  zb::DT         = 600   
-  zi::DT         = 840 
+  q_liq::FT      = 0
+  q_ice::FT      = 0
+  zb::FT         = 600   
+  zi::FT         = 840
+  ziplus::FT     = 875
   dz_cloud       = zi - zb
-  q_liq_peak::DT = 4.5e-4
+  q_liq_peak::FT = 4.5e-4
   
   if xvert > zb && xvert <= zi        
     q_liq = (xvert - zb)*q_liq_peak/dz_cloud
   end
   if ( xvert <= zi)
-    θ_liq  = DT(289)
-    q_tot  = DT(8.1e-3)
+    θ_liq  = FT(289)
+    q_tot  = FT(8.1e-3)
   else
-    θ_liq = DT(297.5) + (xvert - zi)^(DT(1/3))
-    q_tot = DT(1.5e-3)
+    θ_liq = FT(297.5) + (xvert - zi)^(FT(1/3))
+    q_tot = FT(1.5e-3)
   end
 
-  q_pt = PhasePartition(q_tot, q_liq, DT(0))
+  q_pt = PhasePartition(q_tot, q_liq, FT(0))
   Rm    = gas_constant_air(q_pt)
   cpm   = cp_m(q_pt)
   #Pressure
   H = Rm_sfc * T_BL / grav;
   P = P_sfc * exp(-xvert/H);
   #Exner
-  exner_dry = exner(P, PhasePartition(DT(0)))
+  exner_dry = exner(P, PhasePartition(FT(0)))
   #Temperature 
   T             = exner_dry*θ_liq + LH_v0*q_liq/(cpm*exner_dry);
   #Density
   ρ             = P/(Rm*T);
   #Potential Temperature
   θv     = virtual_pottemp(T, P, q_pt)
+    
   # energy definitions
-  u, v, w     = DT(7), DT(-5.5), DT(0)
-  U           = ρ * u
-  V           = ρ * v
-  W           = ρ * w
-  e_kin       = DT(1//2) * (u^2 + v^2 + w^2)
+  u1, u2 = FT(6), FT(7)
+  v1, v2 = FT(-4.25), FT(-5.5)
+  w = FT(0)
+  if (xvert <= zi)
+      u, v = u1, v1
+  elseif (xvert >= ziplus)
+      u, v = u2, v2
+  else
+      m = (ziplus - zi)/(u2 - u1)
+      u = (xvert - zi)/m + u1
+      
+      m = (ziplus - zi)/(v2 - v1)
+      v = (xvert - zi)/m + v1
+  end    
+  e_kin       = FT(1//2) * (u^2 + v^2 + w^2)
   e_pot       = grav * xvert
   E           = ρ * total_energy(e_kin, e_pot, T, q_pt)
   state.ρ     = ρ
-  state.ρu    = SVector(U, V, W) 
+  state.ρu    = SVector(ρ*u, ρ*v, ρ*w) 
   state.ρe    = E
   state.moisture.ρq_tot = ρ * q_tot
 end   
 
 
-function run(mpicomm, ArrayType, dim, topl, N, timeend, DT, dt, C_smag, LHF, SHF, C_drag, zmax, zsponge)
+function run(mpicomm, ArrayType, dim, topl, N, timeend, FT, dt, C_smag, LHF, SHF, C_drag, zmax, zsponge)
 
   grid = DiscontinuousSpectralElementGrid(topl,
-                                          FloatType = DT,
+                                          FloatType = FT,
                                           DeviceArray = ArrayType,
                                           polynomialorder = N,
-                                         )
+                                          )
+  u_ref =  7.0
```

Comment:
```suggestion
  u_ref =  FT(7)
```
likewise for `v_ref`, `w_ref`